cloudify_cli/cli/cfy.py

```diff
@@ -882,6 +895,49 @@ def __init__(self):
             help=helptexts.PASSWORD,
             callback=validate_password)
 
+        self.encryption_password = click.option(
+            '-p',
+            '--password',
+            required=False,
+            help=helptexts.ENCRYPTION_PASSWORD,
+            callback=validate_encryption_password
+        )
+
+        self.not_encrypted = click.option(
+            '--not-encrypted',
+            is_flag=True,
+            required=False,
+            default=False,
+            cls=MutuallyExclusiveOption,
+            mutually_exclusive=['password'],
+            help=helptexts.NOT_ENCRYPTED
+        )
+
+        self.filter_by_visibility = click.option(
+            '-l',
+            '--visibility',
+            required=False,
+            help=helptexts.FILTER_VISIBILITY.format(VisibilityState.STATES)
+        )
+
+        self.filter_by_tenant = click.option(
```

Comment:
Can delete this, we already have an option to filter by tenant, `options.tenant_name`